modules/doc/content/newsletter/2024/2024_10.md

```diff
@@ -1,12 +1,64 @@
 # MOOSE Newsletter (October 2024)
 
-!alert! construction title=In Progress
-This MOOSE Newsletter edition is in progress. Please check back in November 2024
-for a complete description of all MOOSE changes.
-!alert-end!
-
 ## MOOSE Improvements
 
+### Standardization of Physics initialization
+
+For all `Physics` classes deriving from `PhysicsBase`, the variable initialization behavior has been standardized.
+Variables should be initialized using, in order of priority:
+
+- user-specified initialization input parameters, which should always be obeyed
+- from an exodus file, with the `initialize_variables_from_mesh_file` parameter, if no user specified values for a variable are requested
+- from the checkpoint file specified to the `Problem` for the restart
+- from the default value of the initialization parameters, if and only if none of the previous conditions apply
+
+### Direct Nodeset generation
+
+In MOOSE, we often forego the concepts of nodesets and sidesets to simply use the term "boundary", which refers
+to one or the other depending on the objects at hand. For example, a Dirichlet boundary condition for a Lagrange variable
+will use a nodeset for its boundary parameter, while a postprocessor computing the diffusive flux with a monomial variable
+will use a sideset for its boundary parameter. As such, we focused our mesh generation effort to be able to easily create sidesets,
+and the nodesets can be easily generated from these sidesets.
+
+However, for the specific need of generating 1D nodesets within a 3D mesh, we needed to be able to generate nodesets directly.
+The [ParsedGenerateNodeset.md] joins the roster of mesh generators for that purpose. It offers much of the flexibility of
+the [ParsedGenerateSideset.md] and their design are very similar.
+
+### Mesh conversion to all simplices (triangles and tets)
```

Comment:
@roystgnr good enough?

Reply:
Ping @roystgnr

Reply:
Looks good to me; thanks very much for adding it.